janusgraph-dist/docker/test-image.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Copyright 2019 JanusGraph Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+IMAGE=$1
+
+if [ -z "${IMAGE:-}" ]; then
```

Comment:
This doesn't seem to work. I at least get an error if I don't specific an image-name:

```sh
$ janusgraph-dist/docker/test-image.sh
janusgraph-dist/docker/test-image.sh: line 19: $1: unbound variable
```

Reply:
fixed, i justed copied this file.